doc/source/examples/admonitions.rst

```diff
@@ -0,0 +1,31 @@
+.. _admonitions:
+
+Admonitions
+===========
+
+Admonitions are a way to highlight important information in documentation.
+This page shows how the admonitions are rendered in the documentation with ``ansys sphinx theme``.
+
+
+.. jinja:: admonitions
+
+    {% for filename in inputs_admonitions %}
+
+    .. include:: sphinx_examples/{{ filename }}
+
+    {% endfor %}
+
+Credits and copyright
+---------------------
+
+The examples shown above are derived from the PyData Sphinx Theme project, copyrighted by pandas, 2018.
+Original examples can be found at `PyData sphinx theme examples <pydata_examples_>`_.
+These examples are licensed under the BSD 3-Clause License.
+
+Original copyright
+~~~~~~~~~~~~~~~~~~
+
+The examples included here were originally copyrighted and licensed as follows:
```

Comment:
The "included here" is referring to the examples in the PyData Sphinx Theme documentation or in the Ansys Sphinx Theme documentation? Again, I'm not sure why this can't be a paragraph after the examples rather than in multiple sections.

Reply:
examples included in ansys sphinx theme this page.

Reply:
added as `The examples included below in this page were originally copyrighted and licensed as follows:`

Reply:
and moved this sections on the sentence along with admonitions